### ISSUE TITLE
Option to sanitize mail content (pmt_content)

### ISSUE THREAD

**Issue by HexTank** [jenkinsci/poll-mailbox-trigger-plugin#10]
Feature request to sanitize mail content so will always comes through to a job as text/plain.

Something like.
# transform configuration

contentType=plain  (default=asSent)

**Comment by HexTank** [comment]
Thinking about it, it could also be worth sanitizing pmt_replyTo as well, so you can use it direct in ext email plugin, ${pmt_replyTo} in recipients for example, so maybe either having a general sanitize flag in the mailbox plugin settings itself or have

#transform configuration
contentType=plain (default=asSent)
replyTo=plain (default=???)

**Comment by nickgrealy** [comment]
Not quite sure what you want to achieve... do you want access to both the "text/plain" and "text/html" versions of the email? (and why?)

Regarding the pmt_replyTo - I've just added some code (can't find a reference atm) so that instead of "Nick Foobar &lt;nickfoobar@gmail.com&gt;" it just uses "nickfoobar@gmail.com". Checkout 1.022.

**Comment by HexTank** [comment]
No, not both, either one or the other, it's for cases where an email would contain other artefacts other than key value pairs so that some groovy script in a job could do extra bits, but coming through as html, there's a lot more that would need sanitising from the job side, it's a feature more than anything.

Great stuff regarding email.
